provisioning/windows-provision.ps1

```diff
@@ -320,6 +320,24 @@ foreach($k in $downloads.Keys) {
     }
 }
 
+# Special case for Powershell, we need to make sure powershell.exe and pwsh.exe are both available
+# On Windows Server, Windows Powershell 5.1 is installed by default (powershell.exe)
+# On nanoserver, Powershell Core 7 is installed by default (pwsh.ex)
+# https://docs.microsoft.com/en-us/powershell/scripting/whats-new/migrating-from-windows-powershell-51-to-powershell-7?view=powershell-7.2#using-powershell-7-side-by-side-with-windows-powershell-51
+Write-Output "== Ensure both Windows Powershell and Powershell Core are available"
+if ((Get-Host | Select-Object Version).Version.Major -eq 5) {
+    Write-Output "= Windows Powershell already present, installing Powershell Core..."
+    Invoke-Command {& "choco.exe" install pwsh --yes --no-progress --limit-output --fail-on-error-output;}
+    AddToPathEnv "C:\Program Files\PowerShell\7\"
+} else {
+    Write-Output "= Powershell Core already present, installing Windows Powershell..."
+    Invoke-Command {& "choco.exe" install powershell --yes --no-progress --limit-output --fail-on-error-output;}
+    AddToPathEnv "C:\Windows\System32\WindowsPowerShell\v1.0\"
+}
+Write-Output "= Windows Powershell & Powershell Core sanity checks:"
+Invoke-Command {& "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -command "(Get-Host).Version"}
+Invoke-Command {& "C:\Program Files\PowerShell\7\pwsh.exe" -command "(Get-Host).Version"}

```

Comment:
❤️

Reply:
I would have liked to call these without a complete path but didn't managed to get it working.
Tested on a VM where it works without it. 🤷

Reply:
Tried a `refreshenv` in the VM after the `choco install` as recommanded, no luck. Exiting powershell and executing these commands in cmd then worked.
I didn't wanted to loose too much time on this as they won't be called just after this provision script.